service/src/main/webapp/index.html

```diff
@@ -3,7 +3,72 @@
     <head>
         <title>Work Presentation Service</title>
         <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        <base href="http://work-presentation-service.cisterne.svc.cloud.dbc.dk">
+        <script type="text/javascript">
```

Comment:
'Plejer man' ikke at skrive HTML, JavaScript og CSS i hver sin fil? Altså, sådan best practice-wise...

Reply:
Det er et religionsspørgsmål - det er bedst pratice ved større projekter (eg. steder hvor det giver mening at køre minify el. lign. på css/js filerne), men for små sider giver det større netværksbelastning og load tider.